doc/rbd/libvirt.rst

```diff
@@ -95,6 +95,19 @@ To configure Ceph for use with ``libvirt``, perform the following steps:
    **NOTE:** You can also use `rbd create`_ to create an image, but we
    recommend ensuring that QEMU is working properly.
 
+#. **Optional**: Enable libvirt client debugging. If you wish to enable debug
```

Comment:
The "#" is creating a new numeric bullet point for this section. You could also use a directive like ".. tip:: Enable libvirt ..." to create a nicely formatted section.